lua/dap/ext/vscode.lua

```diff
@@ -145,7 +145,12 @@ end
 
 
 function M._load_json(jsonstr)
-  local data = assert(M.json_decode(jsonstr), "launch.json must contain a JSON object")
+  local ok, data = pcall(M.json_decode, jsonstr)
+  if not ok then
+    notify("Error parsing launch.json: " .. jsonstr, vim.log.levels.ERROR)
+    return {}
+  end
+
```

Comment:
```suggestion
  local ok, data = pcall(M.json_decode, jsonstr)
  if not ok then
    error("Error parsing launch.json: " .. data)
  end

```

- Including the whole content is imho too much. `data` will contain the error message if pcall failed which should give a good enough hint ("Expected value but found invalid token at character 1").
- This must still raise an error to not break error suppression in other places. E.g.:

https://github.com/mfussenegger/nvim-dap/blob/bc03b83c94d0375145ff5ac6a6dcf28c1241e06f/lua/dap.lua#L299-L302